gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -774,12 +774,11 @@ message TripDescriptor {
   // In YYYYMMDD format.
   optional string start_date = 3;
 
-  // The relation between this trip and the static schedule. If a trip is done
-  // in accordance with temporary schedule, not reflected in GTFS, then it
-  // shouldn't be marked as SCHEDULED, but likely as ADDED.
+  // The relation between this trip and the trip declared in static data.
   enum ScheduleRelationship {
     // Trip that is running in accordance with its GTFS schedule, or is close
-    // enough to the scheduled trip to be associated with it.
+    // enough to the scheduled trip to be associated with it.  This enumeration must NOT be used for any trip that is
```

Comment:
>The UNSCHEDULED has a comment mentioning that it is experimental and subject to change.

So there are two different schedule_relationship.UNSCHEDULED enumerations:
1. [`TripDescriptor.schedule_relationship.UNSCHEDULED`](https://github.com/google/transit/blob/master/gtfs-realtime/proto/gtfs-realtime.proto#L794-L796) - This field is NOT experimental. This change is related to this field.
2. [`StopTimeUpdate.schedule_relationship.UNSCHEDULED`](https://github.com/google/transit/blob/master/gtfs-realtime/proto/gtfs-realtime.proto#L252-L253) - This field is experimental, but only because we haven't seen anyone publish it in the wild. Conceptually it exists for the exact same reason as `TripDescriptor.schedule_relationship.UNSCHEDULED`.

>This will cause us to lose realtime for all frequency based trips, as we currently don't handle UNSCHEDULED. 

From your response it sounds like there are producers labeling updates for exact_times=0 trips with `schedule_relationship.SCHEDULED` - is that right? If so I'd argue that already goes against the spec - it's pretty clear from the existing spec that those updates should be labeled with UNSCHEDULED. 

That being said, this is the problem we're trying to solve with this PR, so if there are widespread issues with existing producers doing this wrong then we might need a migration path forward to get everyone in-line with the spec. Thoughts on this are welcome.

Reply:
"Trips with ScheduleRelationship=UNSCHEDULED must also set all 
 stopTimeUpdates.ScheduleRelationship=UNSCHEDULED"
It looks non-experimental field suggests to use experimental feed? It is very confusing. Agency setting STU.schedule_relationship = UNSCHEDULED will be surprised that their updates are dropped because we actually don't support this experimental value and require STU.SCHEDULED for such STUs. There is actually a schedule for stop times even for exact_times=0. It just happen that the start of the schedule is shifted to the trip 'start_time' of the trip. I believe agency can express relationship to the schedule in terms of STU.{arrival,departure}.delay. That schedule will also be used if agency haven't specified any STU for a stop and the stop arrival/departure times will be deduced from it.

Reply:
>Agency setting STU.schedule_relationship = UNSCHEDULED will be surprised that their updates are dropped because we actually don't support this experimental value and require STU.SCHEDULED for such STUs. 

How about we put some language here to make it clearer that consumers may choose to continue to process this type of data while producers transition to what the spec says? Some like what's in [TripDescriptor](https://github.com/google/transit/blob/master/gtfs-realtime/spec/en/reference.md#message-tripdescriptor):

"If any enumeration other than UNSCHEDULED is used for exact_times=0 trips it is considered an error and the entity containing the erroneous TripDescriptor may be discarded by consumers."

The "may" here allows you to continue consuming this if you want, even if it doesn't adhere to the spec.

>There is actually a schedule for stop times even for exact_times=0. It just happen that the start of the schedule is shifted to the trip 'start_time' of the trip. I believe agency can express relationship to the schedule in terms of STU.{arrival,departure}.delay. That schedule will also be used if agency haven't specified any STU for a stop and the stop arrival/departure times will be deduced from it.

tl;dr - I've opened an issue [here](https://github.com/google/transit/issues/314) to discuss this specifically - please comment!

For this PR I'd like to stick to clarifying the existing text without getting into those details.